src/averageAges.js

```diff
@@ -15,6 +15,17 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
+  const men = century
+    ? people.filter(person => person.sex === 'm'
+    && Math.ceil(person.died / 100) === century)
+    : people.filter(person => person.sex === 'm');
+
+  const menAgeDied = men.map(year =>
+    year.died - year.born).reduce((sum, x) => sum + x);
+
+  const menAverageAge = menAgeDied / men.length;
```

Comment:
let us move logic of rounding to separate function and call it in all places where needed

Reply:
Thank's! Such kind of solution is more compact))